app.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+$('.checkbox').on('click', function () {
+  $(event.target).toggleClass('unchecked, checked');
```

Comment:
This works great! Instead of listening for a `click`, I'd recommend listening for `input`: that's how you can make sure you get the event to fire, even if the user is using the tab key and spacebar to check/uncheck the checkbox. (Chrome does the right thing, but not all browsers do.)